packages/client-sdk-nodejs/test/integration/integration-setup.ts

```diff
@@ -86,11 +86,20 @@ function sessionCredsProvider(): CredentialProvider {
   return _sessionCredsProvider;
 }
 
+function testAgainstMomentoLocal(): boolean {
+  return process.env.MOMENTO_LOCAL !== undefined;
+}
+
 export function integrationTestCacheClientProps(): CacheClientPropsWithConfig {
+  let credentialProvider = credsProvider();
```

Comment:
do we also want this in the web client?

Reply:
Good question! At the present time we're only using the node client for testing against momento local. In the future we may broaden this configuration feature to all SDKs. For now I'd like to keep it scoped to only what we're using.